Bio/SeqFeature.py

```diff
@@ -927,15 +927,17 @@ def __radd__(self, other):
             return NotImplemented
     
     def __sub__(self,other):
-        """Subtracting an integer will give a new FeatureLocation with
-        its start and end offset shifting back by that amount. For example:
+        """Subtracting an integer will shift back its start and end offset by that amount.
             
+        from Bio.SeqFeature import FeatureLocation
+        >>> f1 = FeatureLocation(5, 10)
+        >>> f2 = FeatureLocation(20, 30)
         >>> print(f1)
         [5:10]
+        >>> print(f1-100)
+        TypeError: unsupported operand type(s) for -: 'FeatureLocation' and 'int'
```

Comment:
The ``TypeError`` is unexpected.

You should make a feature with large location values like 105-150, and then when subtracting 100 your feature should be location 5-50 instead.